tests/errors/reversed_literal_pyobjs.pyx

```diff
@@ -10,22 +10,10 @@ for i in reversed(range(j, [], 2)):
     pass
 for i in reversed(range(j, [], -2)):
     pass
-for i in reversed(range({}, j, 2)):
-    pass
-for i in reversed(range({}, j, -2)):
-    pass
-for i in reversed(range(j, {}, 2)):
-    pass
-for i in reversed(range(j, {}, -2)):
-    pass
 
 _ERRORS = """
 5:24: Cannot coerce list to type 'long'
 7:24: Cannot coerce list to type 'long'
 9:27: Cannot coerce list to type 'long'
 11:27: Cannot coerce list to type 'long'
-13:24: Cannot interpret dict as type 'long'
-15:24: Cannot interpret dict as type 'long'
-17:27: Cannot interpret dict as type 'long'
-19:27: Cannot interpret dict as type 'long'
```

Comment:
I'm taking the view that it's fine if these are a runtime error instead so I've removed them.

Reply:
I think we can still keep the loops, even if they don't raise errors any more. This is quite clearly the only place where such code was exercised. Not all examples in error tests must fail with an error. Some are just there to say "see, the rest is wrong, but this similar code is accepted". We should get notified when that behaviour ever changes, especially by accident.

Reply:
Otherwise, I agree. This isn't something that users necessarily need to be notified at compile time.